tests/test_mgxs_library_condense/results_true.dat

```diff
@@ -1,49 +1,49 @@
-   material  group in nuclide      mean  std. dev.
-0         1         1   total  0.419289    0.01638   material  group in nuclide     mean  std. dev.
-0         1         1   total  0.07774   0.003273   material  group in  group out nuclide      mean  std. dev.
+   group in  material nuclide      mean  std. dev.
```

Comment:
Correct, I believe it is because the derived tallies that are the result of tally arithmetic could have the filters swapped. The values are the same as before, but the ordering of the columns is different hence the new file. That said, some Python 3 tests are failing only because the tally order is different between runs with Python 2 and Python 3. Guess it's now time to learn the differences between Python 2 and Python 3!